conf/provision.yml

```diff
@@ -69,6 +69,21 @@
     - name: restart sshd
       service: name=ssh state=restarted
 
+- name: install go
+  hosts: all
+  user: root
+  tasks:
+    - name: download go binary distribution
+      get_url: >-
+        dest='/tmp'
+        url='https://storage.googleapis.com/golang/go1.5.2.linux-amd64.tar.gz'
```

Comment:
May want to extract the go version that you're locking and put it someplace prominent, and then interpolate here